RUNBOOK.md

```diff
@@ -0,0 +1,88 @@
+- After making changes to the `schema.local` you will need to run `npm run graphql:codegen` to auto-generate new typings and methods for Apollo Client.
+
+### Monaco Editor
+
+- The editor interface itself is implemented using [Monaco Editor](https://microsoft.github.io/monaco-editor/).
+- The editor component can be found here: https://github.com/onflow/flow-playground/tree/master/src/containers/Editor
+- The Cadence language definition (for linting and syntax highlighting) for Monaco can be found here: https://github.com/onflow/flow-playground/blob/master/src/util/cadence.ts
+
+### Cadence Language Server
+
+- The Cadence Language Server (protocol for Monaco) is implemented in Goland and compiled to WASM. 
+  - The WASM bundle is built from [source files in the Cadence repository](https://github.com/onflow/cadence/tree/master/npm-packages/cadence-language-server) and [published on npm](https://www.npmjs.com/package/@onflow/cadence-language-server).
+- You can read more about the Cadence Language Server in the [Cadence repository](https://github.com/onflow/cadence/blob/master/languageserver/README.md).
+- The Playground integration can be found here: 
+  - Server: https://github.com/onflow/flow-playground/blob/master/src/util/language-server.ts
+  - Client: https://github.com/onflow/flow-playground/blob/master/src/util/language-client.ts
+
+## Deployment
+
+The Playground Web App is deployed to [Vercel](https://vercel.com). You will see a link to join the Flow Vercel team when you open your first PR. You must be a member of the team to trigger deployments.
+
+### Staging Deployment
+
+URL: https://play.staging.onflow.org
+
+The Playground Web App is deployed to https://play.staging.onflow.org each time a new commit is pushed to the `staging` branch on this repository.
+
+1. Open a new pull request and select `staging` as the base branch. Vercel will trigger a new deployment once the PR is approved and merged.
+
+2. Vercel will then report the deployment status on the `staging` branch:
+
+![vercel-deployment](vercel-deployment.png)
+
+### Production Deployment
+
+URL: https://play.onflow.org
+
+Once a staging deployment has been verified, you can promote the changes to production.
+
+The Playground Web App is deployed to https://play.onflow.org each time a new commit is pushed to the `production` branch on this repository.
+
+1. Open a new pull request and select `production` as the base branch and `staging` as the source branch. Production deployments should always deploy the same code that is already live in staging. Vercel will trigger a new deployment once the PR is approved and merged.
+
+2. Vercel will then report the deployment status on the `production` branch:
+
+![vercel-deployment](vercel-deployment.png)
+
+## Additional Details
+
+More details and screenshots regarding the deployment process [can be found on Notion](
+https://www.notion.so/dapperlabs/Flow-Playground-Deployment-Process-6ca452adb63a4b41bbe5f3b56eda7021).
```

Comment:
is this doc still needed?

Reply:
No it's not actually. Probably best not to include links to private pages anyways. Thanks!